src/devices/tuya.ts

```diff
@@ -2163,6 +2234,18 @@ const definitions: Definition[] = [
             await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0001', ['_TZ3000_veu2v775']),
+        model: 'TS0001_switch_module_2',
+        vendor: 'TuYa',
+        description: '1 gang mini switch module',
+        whiteLabel: [{vendor: 'MOES', model: 'ZM-104-M'}],
+        extend: tuya.extend.switch({switchType: true}),
```

Comment:
Can be merged with `TS0001_switch_module`? Adding the `tuya.whitelabel` should be enough